examples/post_training/supervised_fine_tune_client.py

```diff
@@ -91,19 +93,100 @@ async def run_main(
 
     print(f"finished the training job: {training_job.job_uuid}")
 
+    if not run_eval:
```

Comment:
❤️  

btw I think these scripts could be unified? cc @raghotham @ashwinb for thoughts on where we want to put these scripts. 
- `llama-stack-client` CLI to trigger post-training run
- this adhoc client script
- notebooks showing same sets of recipes

Reply:
I'd suggest CLI, notebooks, and docs, and not ad hoc code snippets in client package.

Reply:
@yanxi0830 could you elaborate what CLI to trigger post-training run mean here? 

we already have client CLI's to trigger run. But since post training needs to setup several hyper-parameters and put all of them in CLI is not friendly, hence I create this script to make it easier.

Reply:
My understanding is: 

- CLI for post-training could have arguments for filepaths containing hyperparms configs to trigger a post training + eval job (like what this script is doing). 
- Notebooks material is identical to the script to showcase E2E setup. 
- Docs explaining (1) APIs in notebooks to call / commands required to run the CLI.

Reply:
makes sense to me!